build/README.md

```diff
@@ -7,6 +7,8 @@ To build a DOCX file of the manuscript, set the `BUILD_DOCX` environment variabl
 For example, use the command `BUILD_DOCX=true sh build/build.sh`.
 Set a [Travis environment variable](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings) to export DOCX for all Travis builds.
 Currently, equation numbers via `pandoc-eqnos` are not supported for DOCX output.
+Currently, there is varying support for embedding images in DOCX output. Please reference [Pull Request #40](https://github.com/greenelab/manubot-rootstock/pull/40) for possible solutions and continued
```

Comment:
Switch to newline after periods and nowhere else.

Perhaps remove the `Currently,` since it makes the text a bit repetitive.